codegen/Cargo.toml

```diff
@@ -12,19 +12,30 @@ documentation = "https://docs.rs/subxt-codegen"
 homepage.workspace = true
 description = "Generate an API for interacting with a substrate node from FRAME metadata"
 
+[features]
+default = ["fetch_metadata"]
+fetch_metadata = ["dep:jsonrpsee", "dep:tokio", "dep:frame-metadata"]
```

Comment:
I prefer kebab-case as naming convention for optional features i.e, `fetch-metadata` in this case

Reply:
Yep, let's stick with dashes over underscores for consistency :)